snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -145,7 +145,7 @@ open class Snapshot: NSObject {
                 print("Can't prepare environment. Simulator home location is inaccessible. Does \(simulatorHostHome) exist?")
                 return nil
             }
-            homeDir = homeDirUrl
+            homeDir = URL(fileURLWithPath: homeDirUrl.path)
```

Comment:
What you describe would have been my preference as well, the problem is that `fileURLWithPath `doesn't return an optional, so we loose the safety.